netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -284,8 +284,8 @@ public void maxHeaderListSizeShouldBeEnforcedOnServer()  throws Exception {
           + " size limit!");
     } catch (Exception e) {
       Throwable rootCause = getRootCause(e);
-      assertTrue(rootCause.getMessage(),
-          rootCause.getMessage().contains("Header size exceeded max allowed size (1)"));
+      assertTrue(rootCause instanceof StatusException);
```

Comment:
gotcha ... maybe file the netty bug and then add a TODO here that references it?